challenges/linked-list/linked-list.test.js

```diff
@@ -0,0 +1,36 @@
+const LinkList = require('./linked-list.js');
+
+describe('linked lists', () => {
+  it('will construct an empty list', () => {
+    const list = new LinkList();
+    const result = {
+      length: 0,
+      head: null
+    };
+    expect(list).toEqual(result);
+  });
+
+  it('insert a node into list', () => {
+    const list = new LinkList();
+    expect(list.length).toEqual(0);
+    list.insert('node1');
+    expect(list.length).toBe(1);
+    list.insert('node2');
+    expect(list.length).toBe(2);
+  });
+
+  it('node exists in the list', () => {
+    const list = new LinkList;
+    list.insert('node1');
+    expect(list.includes(list.head.num)).toBe(true);
+    expect(list.includes('node2')).toBe(false);
```

Comment:
your tests are passing because of how you have them and your function written, but you need to make changes to the for loop in this method for it to actually work correctly.

Reply:
I would recommend maybe adding several nodes to the list and then testing for includes, instead of only adding one element - the way your function is currently written, it only checks the first element, which is why your tests are passing.